code-challenges.test.js

```diff
@@ -14,42 +14,181 @@
+
+// ● newFib › takes a number of 6 and 10 and returns an array with the Fibonacci Sequence
+
+// ReferenceError: newFib is not defined
+
+//   27 |     const fibLength2 = 10
+//   28 |     // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+// > 29 |     expect(newFib(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+//      |     ^
+//   30 |   expect(newFib(fibLength1)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+//   31 | })
+//   32 | })
+
+//   at Object.expect (code-challenges.test.js:29:5)
+
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+// Snapshots:   0 total
+// Time:        0.157 s
+// Ran all test suites.
+// error Command failed with exit code 1.
+
+
+
+
 // b) Create the function that makes the test pass.
+//PseudoCode
+//input: a number (6)
+//Output: an array within the fibonacci sequence
+//process:Create a function newFibo, add the parameter of an array. 
+//Create conditional statement that ensures number starts with 2
+//create a variable that will store location of new array
+//iterate through the array using for loop
+//take the [i] and create an equation to add the result of the iteration ?
+//create else statement to return "try again"
+
+
+const fibLength1 = 6
+//Expected output: [1, 1, 2, 3, 5, 8]
+
+const fibLength2 = 10
+//Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
+const newFibo = (number) => {
+  if (number < 2)
+  return []
+} for(let i=2; i <Array.length; i++)
+  const fiboSeq = ([i] + fibLength1 -1)
+  
+  consologe log (newFibo(fibLength1))
+
+
 
 // --------------------2) Create a function that takes in an object and returns an array of the values sorted from least to greatest.
 // Hint: Check out this resource: Object.values() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const studyMinutesWeek1 = {
-  sunday: 90,
-  monday: 30,
-  tuesday: 20,
-  wednesday: 15,
-  thursday: 30,
-  friday: 15,
-  saturday: 60
-}
-// Expected output: [15, 15, 20, 30, 30, 60, 90]
+// describe('leastToG', () => {
+//   it('takes an object of studyMinutesWeek1 and 2 and returns an array for each value, sorted least to greatet', () => {
+//     const studyMinutesWeek1 = {
+//       sunday: 90,
+//       monday: 30,
+//       tuesday: 20,
+//       wednesday: 15,
+//       thursday: 30,
+//       friday: 15,
+//       saturday: 60
+//     }  
+//     const studyMinutesWeek2 = {
+//       sunday: 100,
+//       monday: 10,
+//       tuesday: 45,
+//       wednesday: 60,
+//       thursday: 20,
+//       friday: 15,
+//       saturday: 65 }
+//   expect(leastToG(studyMinutesWeek1)).toEqual("[sunday: 100, monday: 10, tuesday: 45, wednesday: 60, thursday: 20, friday: 15, saturday: 65 ]")
+//   expect(leastToG(studyMinutesWeek2)).toEqual("[sunday: 100, monday: 10, tuesday: 45, wednesday: 60 thursday: 20, friday: 15, saturday: 65]")
```

Comment:
Pay attention to the expected output that is given to you by the prompt! It should be an array of all of the values sorted from least to greatest, not a string of an array of the key value pairs of the object like you have in your test.